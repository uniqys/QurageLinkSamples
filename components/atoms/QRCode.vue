<template>
  <canvas class="qrcode">
    QR Code appear here
  </canvas>
</template>

<script lang="ts">
import { Component, Vue } from 'nuxt-property-decorator'

@Component
export default class QRCode extends Vue {
}
</script>
