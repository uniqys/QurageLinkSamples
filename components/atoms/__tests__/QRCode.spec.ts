import { shallowMount } from '@vue/test-utils'
import QRCode from '~/components/atoms/QRCode.vue'

describe('QRCode.vue', () => {
  it('should to match snapshot', () => {
    const data = 'test data'
    const wrapper = shallowMount(QRCode, {
      propsData: {
        data: data
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
