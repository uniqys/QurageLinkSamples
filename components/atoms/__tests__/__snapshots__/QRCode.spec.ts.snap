// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QRCode.vue should to match snapshot 1`] = `
<canvas
  class="qrcode"
  data="test data"
>
  
  QR Code appear here

</canvas>
`;
